osu.Game/Overlays/Profile/Sections/BeatmapsSection.cs

```diff
@@ -21,6 +25,19 @@ public BeatmapsSection()
                 new PaginatedBeatmapContainer(BeatmapSetType.Unranked, User, "Pending Beatmaps"),
                 new PaginatedBeatmapContainer(BeatmapSetType.Graveyard, User, "Graveyarded Beatmaps"),
             };
+
+            foreach (var beatmapContainer in Children.OfType<PaginatedBeatmapContainer>())
+            {
+                beatmapContainer.BeatmapAdded += panel => panel.PreviewPlaying.ValueChanged += isPlaying =>

```

Comment:
this is copy pasting existing logic. the logic should be centralised instead.

why not just expose an event `BeganPlayingPreview` which calls a method `StopPlayingPreview()` on all `PaginatedBeatmapContainer`s before starting the new preview?